src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2622,6 +2622,22 @@ private void InternalSaveAs(string path, SaveContext saveContext, bool isBackup
                 if (!isBackup && hasSaved)
                 {
                     AddToRecentFiles(path);
+
+                    // Track save and save-as operations on workspace, excluding the backup files.
+                    if (saveContext.Equals(SaveContext.Save))
+                    {
+                        Analytics.TrackTaskFileOperationEvent(
+                                  path,
```

Comment:
Can you double check how do we deal with path from analytics, e.g. file open?

Reply:
yes just checked that the filepath shows the whole path and could expose private paths. Should be just track the filename then instead of the whole path?